src/components/Exchange/PositionSeller.js

```diff
@@ -557,16 +664,23 @@ export default function PositionSeller(props) {
     let priceLimit = refPrice.mul(priceBasisPoints).div(BASIS_POINTS_DIVISOR);
     const minProfitExpiration = position.lastIncreasedTime + MIN_PROFIT_TIME;
     const minProfitTimeExpired = parseInt(Date.now() / 1000) > minProfitExpiration;
+
     if (nextHasProfit && !minProfitTimeExpired && !isProfitWarningAccepted) {
       if ((position.isLong && priceLimit.lt(profitPrice)) || (!position.isLong && priceLimit.gt(profitPrice))) {
         priceLimit = profitPrice;
       }
     }
+    const path = [tokenAddress0];
 
-    const withdrawETH = collateralTokenAddress === AddressZero || collateralTokenAddress === nativeTokenAddress;
+    if ((receiveToken?.address !== tokenAddress0) && allowReceiveTokenChange) {
```

Comment:
in case `receiveToken` is undefined that condition can be true. and on next line it will fail? because it will try to read property of undefined